pkg/trie/triedb/triedb.go

```diff
@@ -774,7 +774,8 @@ func (t *TrieDB) commit() error {
 				switch n := node.(type) {
 				case newNodeToEncode:
 					hash := common.MustBlake2bHash(n.value)
-					err := dbBatch.Put(hash[:], n.value)
+					prefixedKey := bytes.Join([][]byte{n.partialKey, hash.ToBytes()}, nil)
```

Comment:
I think the append syntax is easier to read if you're passing `nil` as the seperator.

Reply:
I made it this way to match with [other part of our code](https://github.com/ChainSafe/gossamer/blob/33758dde9647cea5a69621c20f90c832985acce6/pkg/trie/triedb/triedb.go#L851-L853) that is doing something similar

Reply:
I think in both cases it's easier to read using append syntax.

Reply:
Changed